### PR TITLE
tkt-52507:  Expose --transfers rclone option

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: gui/tasks/forms.py [hunk 1]
```diff
@@ -152,6 +161,16 @@ def clean_dayweek(self):
         w = ','.join(w)
         return w
 
+    def clean_transfers(self):
+        value = self.cleaned_data.get('transfers')
+        if value:
+            try:
+                return int(value)
```
Comment (william-gr) [inline gui/tasks/forms.py]: If all it can be is an integer, why are we using a CharField?

Comment (themylogin) [reply gui/tasks/forms.py]: I've did it multiple times because django does not have an `IntegerField` that allows empty input.

Comment (william-gr) [reply gui/tasks/forms.py]: Thats not django, its dojo as far as I know.
Have you tried using a TextInput widget with IntegerField formfield?

Comment (themylogin) [reply gui/tasks/forms.py]: Yes, `IntegerField` always tries to do `int(value)`

Comment (themylogin) [reply gui/tasks/forms.py]: Thank you, it worked!

**Pull request merged**